test/test_invalid_headers.py

```diff
@@ -282,11 +285,11 @@ def test_invalid_pseudo_headers(self, hdr_validation_flags):
 
     @pytest.mark.parametrize('validation_function', validation_functions)
     @pytest.mark.parametrize(
-        'hdr_validation_flags', hdr_validation_no_trailers
+        'hdr_validation_flags', hdr_validation_request_headers_no_trailer
     )
-    def test_matching_authority_host_headers(self,
-                                             validation_function,
-                                             hdr_validation_flags):
+    def test_matching_authority_host_headers(
+        self, validation_function, hdr_validation_flags
+    ):
```

Comment:
Heh, sorry, I know that this isn't what you were told earlier, but the original style of this function was better. Basically, for function definitions I like having the argument names lined up, but for any other multiline sections I like having the parentheses by themselves.